src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -582,6 +595,7 @@ private function addAssetsSection(ArrayNodeDefinition $rootNode)
                                         ->end()
                                     ->end()
                                     ->scalarNode('version_format')->defaultNull()->end()
+                                    ->scalarNode('json_manifest_path')->defaultNull()->end()
```

Comment:
Just asking: would `manifest_path` be a more future-proof name than `json_manifest_path` ? Or is this too much future thinking? Can we expect to support other formats different than JSON?